docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,589 @@
+
+# Load the SIAP metric
+from stonesoup.metricgenerator.tracktotruthmetrics import SIAPMetrics
+from stonesoup.measures import Euclidean
+
+siap_kf_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                            velocity_measure=Euclidean((1, 3)),
+                            generator_name='SIAP_KF_fused-truth',
+                            tracks_key='KF_fused_tracks',
+                            truths_key='truths'
+                            )
+siap_kf1_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                             velocity_measure=Euclidean((1, 3)),
+                             generator_name='SIAP_KF1-truth',
+                             tracks_key='KF_1_tracks',
+                             truths_key='truths'
+                             )
+siap_kf2_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                             velocity_measure=Euclidean((1, 3)),
+                             generator_name='SIAP_KF2-truth',
+                             tracks_key='KF_2_tracks',
+                             truths_key='truths'
+                             )
+
+siap_pf_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                            velocity_measure=Euclidean((1, 3)),
+                            generator_name='SIAP_PF_fused-truth',
+                            tracks_key='PF_fused_tracks',
+                            truths_key='truths'
+                            )
+siap_pf1_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                             velocity_measure=Euclidean((1, 3)),
+                             generator_name='SIAP_PF1-truth',
+                             tracks_key='PF_1_tracks',
+                             truths_key='truths'
+                             )
+
+siap_pf2_truth = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                             velocity_measure=Euclidean((1, 3)),
+                             generator_name='SIAP_PF2-truth',
+                             tracks_key='PF_2_tracks',
+                             truths_key='truths'
+                             )
+
+# Define a data associator between the tracks and the truths
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+associator = TrackToTruth(association_threshold=30)
+
+from stonesoup.metricgenerator.manager import MultiManager
+metric_manager = MultiManager([basic_KF,
+                               basic_KF1,
+                               basic_KF2,
+                               basic_PF,
+                               basic_PF1,
+                               basic_PF2,
+                               siap_kf_truth,
+                               siap_kf1_truth,
+                               siap_kf2_truth,
+                               siap_pf_truth,
+                               siap_pf1_truth,
+                               siap_pf2_truth],
+                              associator)
```

Comment:
This section is quite chunky, could you move this metrics setup code into Section 4 with the metrics evaluation?